src/widgets/configure/configure.js

```diff
@@ -29,23 +30,12 @@ Full documentation available at https://community.algolia.com/instantsearch.js/v
  *   })
  * );
  */
-export default function configure(searchParameters = {}) {
-  if (!isPlainObject(searchParameters)) {
+export default function configure(searchParameters) {
+  try {
+    // make it clear we do not have default renderFn && unmountFn for this widget
+    const makeWidget = connectConfigure(undefined, undefined);
```

Comment:
I like the explicitness, but isn't a bit redundant? Why not `connectConfigure()`